front/src/reducers/osrdconf/index.ts

```diff
@@ -209,13 +211,14 @@ export default function reducer(inputState: OsrdMultiConfState | undefined, acti
         if (draft[section].originLinkedBounds) {
           const difference =
             draft[section].originTime && draft[section].originUpperBoundTime
-              ? time2sec(draft[section].originUpperBoundTime) - time2sec(draft[section].originTime)
+              ? time2sec(draft[section].originUpperBoundTime as string) -
```

Comment:
I think it's better if you get the var `originLinkedBounds, originTime, originUpperBoundTime` from the `draf[section]`, this way Typescript knows clearly whether they are undefined or not and this clarify the code (and you can remove all the casts)
```suggestion
  case UPDATE_ORIGIN_TIME: {
        const newOriginTimeSeconds = time2sec(action.originTime);
        const { originLinkedBounds, originTime, originUpperBoundTime } = draft[section];
        if (originLinkedBounds) {
          const difference =
            originTime && originUpperBoundTime
              ? time2sec(originUpperBoundTime) - time2sec(originTime)
              : ORIGIN_TIME_BOUND_DEFAULT_DIFFERENCE;
          draft[section].originUpperBoundTime = sec2time(newOriginTimeSeconds + difference);
        }
```

Reply:
ok